src/wui/http/http_client.c

```diff
@@ -435,32 +437,25 @@ err_t data_received_fun(void *arg, struct tcp_pcb *tpcb, struct pbuf *p, err_t e
     }
     if(request_part[0] == '{'){
         http_json_parser((char *)&request_part, len_copied);
-    } else if (request_part[0] == 'G' || request_part[0] == 'M'){
+    } else {
         http_lowlvl_gcode_parser((char *)&request_part, len_copied, command_id);
     }
     return ERR_OK;
 }
 
-static const char * telemetry_to_http_str(const char * host_ip4_str){
-    char header[HEADER_MAX_SIZE];
-    char *uri = "/p/telemetry";
-    char printer_token[API_TOKEN_LEN + 1]; // extra space of end of line
-
-    eeprom_get_string(EEVAR_CONNECT_KEY_START, printer_token, API_TOKEN_LEN);
-    printer_token[API_TOKEN_LEN] = 0;
-    snprintf(header, HEADER_MAX_SIZE, "POST %s HTTP/1.0\r\nHost: %s\nPrinter-Token: %s\r\n", uri, host_ip4_str, printer_token);
-    
-    return get_update_str(header);
-}
-
-static const char * events_to_http_str(void * container){
-    
-    char printer_token[API_TOKEN_LEN + 1]; // extra space of end of line
-    char header[HEADER_MAX_SIZE];
+static void create_http_header(char * dest, const char * method, const char * uri, const char * content_type, const char * host_ip){
```

Comment:
investigate if possible to use same as in http server requests, or POST requests